### PR TITLE
Quick start revamp 1.4.0

### PR DESCRIPTION
**Pull request by brockfanning**
These are documentation updates to get a simpler quickstart. These depend on two other PRs:
* https://github.com/open-sdg/open-sdg-site-starter/pull/75
* https://github.com/open-sdg/jekyll-open-sdg-plugins/pull/102

Since there a lot of changes here are links directly to the new versions of the 3 files:
* https://github.com/brockfanning/open-sdg/blob/quick-start-revamp-1-4-0/docs/quick-start.md
* https://github.com/brockfanning/open-sdg/blob/quick-start-revamp-1-4-0/docs/automation/github.md
* https://github.com/brockfanning/open-sdg/blob/quick-start-revamp-1-4-0/docs/automation/triggered-site-builds.md

### CODE REVIEW

File: docs/quick-start.md [hunk 1]
```diff
@@ -2,63 +2,64 @@
...
-> the [open-sdg-simple-starter project](https://github.com/open-sdg/open-sdg-simple-starter).
-
 ## Signing up and creating repositories
 
 1. If you don't already have a Github.com account, [go to Github.com](https://github.com/) to sign up and then log in.
 1. Go to the [site starter](https://github.com/open-sdg/open-sdg-site-starter) and click the green "Use this template" button.
-1. Next you will be prompted to choose a name for your new repository. This will affect the URL at which you access the site later, so choose carefully. A suggestion might be: `sdg-site-australia` (adjusted for your country). Note that you can change this later if needed.
-1. Enter a description if you would like. Leave "Public" selected, check the "Include all branches" box, and click "Create repository from template".
-    * Bookmark the created repository -- this is your "__site repository__".
+1. You can enter any name for the repository. Here we recommend using "site".
```
Comment (LucyGwilliamAdmin) [inline docs/quick-start.md:9]: @brockfanning I thought that they would have to name the repos open-sdg-site-starter and open-sdg-data-starter in order for the site to build without any changes being needed?

Comment (LucyGwilliamAdmin) [reply docs/quick-start.md:9]: Ah I see - you have a section below about connecting the site and data

Comment (LucyGwilliamAdmin) [reply docs/quick-start.md:9]: Sounds great!

**Review by LucyGwilliamAdmin — APPROVED** [review]
Approving as content looks good but will be asking James G to test it once all changes are finalised - think we could open a new PR if changes are needed as result of that though?

**Comment by brockfanning** [comment]
Sounds good, I'll merge the other PRs and this one.

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin One note about testing the updated quick-start - because our starters are using a "1.4.0-dev" branch, I think an extra step will be needed at the beginning, to merge the "1.4.0-dev" branches into the "develop" branches.

**Pull request merged**